src/movies.cpp

```diff
@@ -349,6 +349,8 @@ void ff8_prepare_movie(uint32_t disc, uint32_t movie)
 			if(res > 0) camOffset += res;
 		}
 
+		fclose(camFile);
```

Comment:
Shouldn't this be inside the `if`? If you're on disk 4 this code will crash as it is.

Reply:
Which if? Right now I'm closing it after it has been read inside the while

Reply:
The if block just above this line. Where the `fopen` happens :)

Reply:
If you are talking about the if block on line 339, that enters only if the file is not opened correctly. I don't think there is the need to close it when it is not opened

Reply:
Can you just drop this commit for now and we can re-evaluate later? Thanks!

Reply:
Done